org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/DTDCompletionExtensionsTest.java

```diff
@@ -196,7 +196,7 @@ public void externalDTDCompletionAllDeclsSnippetsNotSupported() throws BadLocati
 	public void testNoDuplicateCompletionItems() throws BadLocationException {
 		// completion on <|
 		String xml = "<?xml version=\"1.0\" standalone=\"no\" ?>\n"
-				+ "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.0//EN\" \"http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg.dtd\">\n"
+				+ "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.0//EN\" \"http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg.dtd\">\n" // FIXME: I can't find this file. does it still exist?

```

Comment:
Don't change this line, this test uses XML catalog and test is working (see CI build)

Reply:
yes, that was just a temporary "note to self" :)